packages/extension-ui/src/components/Address.test.tsx

```diff
@@ -13,10 +13,13 @@ import React from 'react';
 import { act } from 'react-dom/test-utils';
 import { ThemeProvider } from 'styled-components';
 
+import allChains from '@polkadot/extension-chains/chains';
+import { Chain } from '@polkadot/extension-chains/types';
```

Comment:
Import type { ... }

Reply:
Thanks, I'll do a cleanup soon, hope to finish this suite of tests today